napi-inl.h

```diff
@@ -425,6 +425,38 @@ inline std::u16string String::Utf16Value() const {
 // Object class
 ////////////////////////////////////////////////////////////////////////////////
 
+namespace helper {
+
+template <typename Key>
+class PropertyLValue {
```

Comment:
’xactly. :smile: Happy to do whatever you prefer, but as long as it’s not intended for public usage it won’t matter much anyway.